Essentials/src/main/java/com/earth2me/essentials/Essentials.java

```diff
@@ -125,6 +126,7 @@
     private transient PermissionsHandler permissionsHandler;
     private transient AlternativeCommandsHandler alternativeCommandsHandler;
     private transient UserMap userMap;
+    private transient BalanceTopImpl balanceTop;
```

Comment:
shouldn't the type be the interface and not the impl?

Reply:
`Essentials` is an impli class itself so it doesn't really matter. I'll keep it in case we need to use any impli methods in the Ess class itself.